azext_iot/digitaltwins/params.py

```diff
@@ -433,3 +433,91 @@ def load_digitaltwins_arguments(self, _):
     with self.argument_context("dt network private-endpoint connection wait") as context:
         context.ignore("created")
         context.ignore("exists")
+
+    with self.argument_context("dt data-history") as context:
+        context.argument(
+            "conn_name",
+            options_list=["--conn-name", "--cn"],
+            help="Name of time series database connection.",
+            arg_group="TimeSeries Connection",
```

Comment:
This arg_group should be removed, conn_name is a required fundamental arg for the command and its implied at the instance level. Separating into a different arg group muddies the presentation IMO.